tests/api/test_bulk_delete.py

```diff
@@ -387,12 +386,35 @@ def test_delete_menu_items(staff_api_client, menu_item_list, permission_manage_m
     )
     content = get_graphql_content(response)
 
-    assert content["data"]["menuItemBulkDelete"]["count"] == 3
+    assert content["data"]["menuItemBulkDelete"]["count"] == len(menu_item_list)
     assert not MenuItem.objects.filter(
         id__in=[menu_item.id for menu_item in menu_item_list]
     ).exists()
 
 
+def test_delete_empty_list_of_ids(staff_api_client, permission_manage_menus):
+    query = """
+    mutation menuItemBulkDelete($ids: [ID]!) {
+        menuItemBulkDelete(ids: $ids) {
+            count
+        }
+    }
+    """
+    menu_item_list = []
+    variables = {
+        "ids": [
+            graphene.Node.to_global_id("MenuItem", menu_item.id)
+            for menu_item in menu_item_list
```

Comment:
So if `menu_item_list` is empty, then maybe we should explicitly use empty list?
Or `"ids": menu_item_list` ?